compiler/llvm/clangUtil.cpp

```diff
@@ -1496,12 +1497,47 @@ void setupClang(GenInfo* info, std::string mainFile)
 
   std::unique_ptr<clang::driver::Compilation> C(TheDriver.BuildCompilation(clangArgs));
 
-  INT_ASSERT(C->getJobs().size() == 1);
+  clang::driver::Command* job = NULL;
+
+  if (localeUsesGPU() == false) {
+    // Not a CPU+GPU compilation, so just use first job.
+    job = &*C->getJobs().begin();
+  } else {
+    // CPU+GPU compilation
+    //  1st cc1 command is for the GPU
+    //  2nd cc1 command is for the CPU
+    for (auto &command : C->getJobs()) {
+      bool isCC1 = false;
+      for (auto arg : command.getArguments()) {
+        if (0 == strcmp(arg, "-cc1")) {
+          isCC1 = true;
+          break;
+        }
+      }
+      if (isCC1) {
+
+        if (gCodegenGPU) {
+          // For GPU, set j to 1st cc1 command
```

Comment:
I think this and the other comment should say "job" instead of "j"?  It looks like "j" was the old name of the variable